torch/_subclasses/fake_tensor.py

```diff
@@ -1378,7 +1389,8 @@ def _cached_dispatch_impl(
                 if self.cache_crosscheck_enabled:
                     # For debugging / testing: Validate that the output synthesized
                     # from the cache matches the output created by normal dispatch.
-                    self._crosscheck_cache_output(output, func, types, args, kwargs)
+                    with disable_fake_tensor_cache(self):
```

Comment:
btw, when is self.cache_crosscheck_enabled set to True? I assume it is set to True somewhere in testing which is why this change was necessary

Reply:
Yes, in test_fake_tensor.py with 

```torch._dynamo.config.fake_tensor_cache_crosscheck_enabled = True```